MediaManager.Android/Video/VideoPlayerImplementation.cs

```diff
@@ -386,5 +387,24 @@ public bool OnInfo(MediaPlayer mp, [GeneratedEnum] MediaInfo what, int extra)
 
             return true;
         }
+
+        private string GetFullPathUrl(string url)
+        {
+            if (url.StartsWith("http", StringComparison.OrdinalIgnoreCase)
+                || url.StartsWith("android.resource", StringComparison.OrdinalIgnoreCase))
+            {
+                return url;
+            }
+            else if (!System.IO.Path.IsPathRooted(url))
+            {
+                string file = System.IO.Path.GetFileNameWithoutExtension(url);
+                string package = Application.Context.PackageName;
+                return $"android.resource://{package}/raw/{file}";
```

Comment:
Think twice about this, you expect people to have a "raw" folder in their resources before they can play local content?

I would make use of the IMediaFile.ResourceAvailability instead.